server/game/cards/02-AoA/PhloxemSpike.js

```diff
@@ -4,7 +4,7 @@ class PhloxemSpike extends Card {
     setupCardAbilities(ability) {
         this.play({
             effect: 'destroy each creature not on a flank.',
-            condition: () => this.controller.creaturesInPlay.length === 0,
+            condition: (context) => context.game.activePlayer.creaturesInPlay.length === 0,
```

Comment:
This should probably be `context.player`, although I can't think of any situation where it might matter....

Reply:
Fixed